src/main/java/uk/gov/companieshouse/presenter/account/controller/PresenterAccountController.java

```diff
@@ -1,22 +1,24 @@
 package uk.gov.companieshouse.presenter.account.controller;
 
-import java.net.URI;
-
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.http.converter.HttpMessageNotReadableException;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
-
 import uk.gov.companieshouse.logging.Logger;
 import uk.gov.companieshouse.presenter.account.exceptionhandler.ValidationException;
-import uk.gov.companieshouse.presenter.account.model.request.presenter.account.PresenterRequest;
+import uk.gov.companieshouse.presenter.account.model.PresenterAccountDetails;
+import uk.gov.companieshouse.presenter.account.model.request.PresenterAccountDetailsRequest;
 import uk.gov.companieshouse.presenter.account.service.PresenterAccountService;
 
+import java.net.URI;
+import java.util.Optional;
```

Comment:
Shouldn't these be at the top for alphabetical order?

Reply:
Yeah. I had my code style slightly wrong so it was trying to put `java` and `javax` imports below all others.
I've fixed that now and mved the imports.